2-mandatory/7-eligible-students.js

```diff
@@ -14,7 +14,11 @@
   - Returns an array containing only the names of the who have attended AT LEAST 8 classes
  */
 
-function eligibleStudents(attendances) {}
+function eligibleStudents(attendances) {
+  return attendances
+    .filter((attendance) => attendance.attendance >= 8)
+    .map((name) => name.name);
+}
 
```

Comment:
Great implementation of .filter and .map methods. I have used a for in statement and an if statement to achieve the same solution. Both work great.